package.json

```diff
@@ -395,6 +395,9 @@
     "elliptic": "6.5.4",
     "serialize-to-js": "3.1.1",
     "netmask": "2.0.1",
-    "ssri": "8.0.1"
+    "ssri": "8.0.1",
+    "hosted-git-info": "3.0.8",
+    "underscore": "1.12.1",
+    "lodash": "4.17.21"
```

Comment:
I believe you don't need the resolution for lodash since you upgrade the dependency above

Reply:
Without it:
```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ high          │ Command Injection                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.21                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ @nestjs/swagger                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ @nestjs/swagger > lodash                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1673                        │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ high          │ Command Injection                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.21                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ @nestjs/graphql                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ @nestjs/graphql > lodash                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1673                        │
└───────────────┴──────────────────────────────────────────────────────────────┘

```